posthog/src/main/java/com/posthog/PostHog.kt

```diff
@@ -466,6 +469,35 @@ public class PostHog private constructor(
         }
     }
 
+    /**
+     * Because we use UUIDv7
+     * we can convert a session id into the session start timestamp
+     * and then use that to enforce the maximum session length
+     */
+    private fun enforceMaxSessionLength(buildProperties: Map<String, Any>): Map<String, Any> {
```

Comment:
react native has its own session id rotation so we should not do this for RN I think
https://github.com/PostHog/posthog-js-lite/blob/b8372284df5489457d95b16b0e4b12a91dd66c26/posthog-core/src/index.ts#L1309-L1315
otherwise they will be out of sync (RN uses a different session id than Android)

Reply:
so, needs to be configurable? and then RN would turn off the Android one?
or sync up a release and RN stops doing its own?